app/models/user.rb

```diff
@@ -1,26 +1,31 @@
 class User < ApplicationRecord
   has_secure_password
 
+  USER_PERMIT = %i(name email password password_confirmation date_of_birth
+gender).freeze
+
   VALID_EMAIL_REGEX = Regexp.new(Settings.development.user.valid_email_regex,
                                  Regexp::IGNORECASE)
   PASSWORD_REQUIREMENT =
     Regexp.new(Settings.development.user.password_requirement)
 
-  validates :name, presence: true,
+  validates :name, presence: {message: :cant_be_blank},
```

Comment:
```suggestion
  validates :name, presence: true,
```
E dùng key mặc định, k custom thêm messages key với cho các validates của rails hỗ trợ
Fix tương tự cho những chỗ khác
https://guides.rubyonrails.org/i18n.html#error-message-interpolation